build-scripts/licensechecker/licensewhitelist.json

```diff
@@ -11,6 +11,7 @@
     "Apache*",
     "Unlicense",
     "Zlib",
+    "CC0-1.0",
```

Comment:
confirmed ok?

Reply:
I'm not sure who to confirm this with! The license seems fine?